### DISCUSSION TITLE
Is version author determined during tag ? not export ?

### DISCUSSION THREAD

**Discussion by p-l-l** [teambit/bit#4666] [Help] [answered]
Hi, I recently exported some components to bit.dev and got an unexpected result so I am
asking this question before considering this as bug vs expected behavior....

The author of the v0.0.1 of my components were not the one I was logged in for the export.
The change log of the exported components shows a user that has no permission on the
collection which feels a bit odd to me...

### Context and additional information

First of all, I used to be logged in to bit with my personnal account that I used to learn the basics of Bit...
Then I made a new account on behalf of my employer and made a private organisation with that account.
I forgot to logout from bit in my terminal, and coded a few things, tagged them, and when I ran the export
command I got a permission denied, which is fine since collection is private to the other account.

At this point, I logged out from my personnal account, logged in on the employer account and ran the export
command. Here is the result:

These are the permissions of the private collection... the skytech-admin is the only member and can't add collaborators.
![image](https://user-images.githubusercontent.com/23360637/127645897-a26a6dd3-dc68-445a-bfaf-2f33fc386e34.png)

Here is an example of change log of one of the component's
![image](https://user-images.githubusercontent.com/23360637/127646018-db4507f0-8490-4d96-93bc-1b9353bbd247.png)

I was expecting to see skytech-admin as the one who released v0.0.1, but I would understand that in some other context, exporting could be done by a bot..

### Update
I noticed that the user Pierreluclemire is shown by Bit doctor, even if im logged in as another account (with bit login). Is this normal behavior ? it could explain why the author of tag is not the one im logged in as on my IDE with bit login...

**Comment by itaymendel** [comment, marked as answer]
Sorry for the delayed response.

First of all, yes - everything about the version happens during the `tag` process. Including setting the author.
To set author information Bit does the following:
1. Take `user.name` and `user.email` from `bit config`
2. If information not found - use `git config` to fetch author information.
3. In Bit.dev we try to correlate this information with a registered user and if we can - we link them.

I hope this answers to you question.
